radix-engine/src/blueprints/models/native_blueprint_state_macro.rs

```diff
@@ -0,0 +1,927 @@
+use crate::internal_prelude::*;
+
+/// Generates types and typed-interfaces for native blueprints, their
+/// state models, features, partitions, and their interaction with
+/// the substate store.
```

Comment:
Update comment to remove "partitions and their interactions with the substate store"